src/main.ts

```diff
@@ -299,18 +164,18 @@ function handleFeatureChange({ currentTarget }: Event): void {
   updateFeature(value);
 }
 
-function updateFeature(newFeatureName: string): void {
+async function updateFeature(newFeatureName: string): Promise<void> {
```

Comment:
I forget, isn't it the case if your function is marked `async` then the Promise on the return value is implied?

Reply:
Doesn't seem like it! This is the error message that the typescript checker spits out if I do that:
`TS1064: The return type of an async function or method must be the global Promise<T> type. Did you mean to write 'Promise<void>'?`